CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Update list match confirm / rejection to be done by admins [#2030](https://github.com/open-apparel-registry/open-apparel-registry/pull/2030)
 - Update moderation UI to trigger batch processing [#2048](https://github.com/open-apparel-registry/open-apparel-registry/pull/2048)
 - Create a log group for AWS Batch jobs [#2060](https://github.com/open-apparel-registry/open-apparel-registry/pull/2060)
+- Replace font and global colors [#2059](https://github.com/open-apparel-registry/open-apparel-registry/pull/2077)
```

Comment:
Traditionally we have uses the PR number as the link text in our CHANGELOG entries. Consider changing 2059 to 2077